collectors/python.d.plugin/chrony/README.md

```diff
@@ -41,6 +55,7 @@ local:
   command: 'chronyc -n tracking'
 ```
 
----
+Save the file and restart the Netdata Agent with `service netdata restart`, or the appropriate method for your system,
```

Comment:
@joelhans like I said in another issue, we need to start using `systemctl` the syntax is more or less the same:

```bash
$ sudo systemctl restart netdata
```

Reply:
@thiagoftsm Okay, I think I've misunderstood the purpose of `service` all along. I'll change it in my handful of recent PRs, but I'll need to embark on a larger project to change the usage in many more places.